Add optional preview to generators (#587)
github/view_component: CHANGELOG.md

## main

> * Add `--preview` option to the generator. Generates associated preview file.

Reviewer: ```suggestion
* Add `--preview` generator option to create an associated preview file.
```
